homeassistant/components/nest/api.py

```diff
@@ -94,16 +94,14 @@ async def async_get_access_token(self) -> str:
 
     async def async_get_creds(self) -> Credentials:
         """Return an OAuth credential for Pub/Sub Subscriber."""
-        return Credentials(  # type: ignore[no-untyped-call]
+        return Credentials(
```

Comment:
Looks like MyPy disagrees with this change
```suggestion
        return Credentials(  # type: ignore[no-untyped-call]
```